frontend/src/App.js

```diff
@@ -411,6 +412,90 @@ export default function App() {
                   component={() => <RoutineReports />}
                   role="Reports"
                 />
+                <SecureRoute
+                  path="/PatientStatusReport"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/StatisticsReport"
```

Comment:
Looks like you have changes that dont belong to your PR , can you only leave changes that are specific for the report youre working on ??
i see you have some changes from this PR https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/740

Reply:
Hey @mozzy11 for your kind information :slightly_frowning_face: there is conflict because PR where not merged in series you should from below to top the line. Also i didn't copied from PR #740 might he copied from my PR #714 and its routing code because i raised this PR first as u can see the number difference. :slightly_smiling_face: 

Thank You :100:

Reply:
yeah i was unsure that why i have added **might** there :slightly_smiling_face:  ! Am not blaming any on here.
I hope my words getting transferred well to @manishjha-04 although he is my good frnd.

Reply:
Hello @adityadeshlahre,

I hope you're doing well! 💫 I just wanted to share a perspective regarding our codebase. In the spirit of open source, it's all about continuous learning and enhancing our codebase together. 🚀

I think what @mozzy11 meant was that we should implement the route on the reports we are working with. You have implemented extra routes for other reports too. This way, we can work more effectively, making sure everything is consistent and works well. 🛠️

Let's keep collaborating positively, leveraging each other's strengths to improve our project further. 💪